### DISCUSSION TITLE
Add More User Settings

### DISCUSSION THREAD

**Discussion by PJGitHub9** [sct/overseerr#2365] [Feature Requests]
I just recently switched from Ombi to Overseer and I'm loving the change so far. The only thing I would like to see is a change in the user settings. If possible, I would like to see an option for local users to not use emails. I only use overseer for my family, and Ombi had a feature which let me set a username and simple password for family members. So the username would just be our last name, and the password was something simple they could remember. 

TL;dr; Users should be able to just use a username, and set your own password requirements.
